.github/workflows/continuous-integration-workflow.yml

```diff
@@ -26,6 +26,8 @@ jobs:
       docker-tag: ghcr.io/kokkos/ci-containers/${{ matrix.config.dockerfile }}:${{ matrix.config.tag }}
     runs-on: ubuntu-latest
     steps:
+      - name: Checkout CI repo
+        uses: actions/checkout@v4
```

Comment:
Why did we add this?

Reply:
Now we need to checkout the Dockerfiles as we modify them below. Previously the docker action did that for us automatically.